rfcs/064-graph-data-model/README.md

```diff
@@ -0,0 +1,129 @@
+# RFC 064: Graph data model
+
+## Context
+
+Following from [062-knowledge-graph](../062-knowledge-graph/README.md), which provides an overview of a past prototype and proposal for a new graph based on data from the Collection and external ontologies, this RFC focuses on a new graph data model. While the primary/current focus of the suggested model is the enrichment of concepts, other data and attributes are also included as these may enable future machine learning work and visualisation. The graph data model is designed to enable enrichment of concepts while keeping original labels and sources intact. This is to make sure we are always able to tell where a new link or piece of information was acquired from, whether it be manual tagging, a particular source ontology, or ML inference.
+
+## Overview
+
+Here is a visual overview of the proposed data model:
+
+![graph_model](figures/graph_model.png)
+
+## Nodes
+
+### Work
+
+Properties on `Work` nodes can be found in [work.yaml](yaml_files/work.yaml). These are derived from information available in the works snapshot. There are several properties which could be linked directly to source nodes in the future, such as language and production location. Attributes containing potentially useful text and descriptions are all included. Any free text which provides useful information about the work may be relevant for future machine learning work. This includes title, description, physical description, notes, lettering.
+
+### Series
+
+Series nodes only have a `title` property (see [series.yaml](yaml_files/series.yaml)). These are extracted from works and their main purpose is to represent meaningful links between different works which are part of the same series.
+
+### Image
+
+Nodes for images which are part of works. Suggested attributes can be viewed in [image.yaml](yaml_files/image.yaml), and include information such as the identifier and iiif url.
```

Comment:
Images in our [images API](https://api.wellcomecollection.org/catalogue/v2/images?include=source.subjects) do have the subjects of the works they are extracted from on them. If we are intentionally omitting that relationship, it's worth mentioning it.